apps/omg_watcher_rpc/test/omg_watcher_rpc/web/response_test.exs

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.ResponseTest do
+  # async: false because it needs to manipulate the global :api_mode application env.
+  use ExUnit.Case, async: false
+  alias OMG.WatcherRPC.Web.Response
+
+  @app :omg_watcher_rpc
+
+  setup do
+    original_mode = Application.get_env(@app, :api_mode)
+
+    on_exit(fn ->
+      _ = Application.put_env(@app, :api_mode, original_mode)
+    end)
+  end
+
+  describe "add_app_infos/1" do
+    test "appends the given map with a service_name and version" do
+      :ok = Application.put_env(@app, :api_mode, :watcher)
+      assert %{foo: "bar", service_name: _, version: _} = Response.add_app_infos(%{foo: "bar"})
```

Comment:
how about testing if the `version` field holds in fact a version as in [here](https://github.com/omisego/elixir-omg/pull/1266/commits/19063bd63ace82b248f9794a10c319b4361405f2#diff-3c6511e75122d06d1b190c6548d3af09R30)

> `assert {:ok, _} = Version.parse(conn.assigns.app_infos.version)`

?

Reply:
Ah I missed this. Added in 5b3ef09 and b88a2c5.